src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -5,7 +5,143 @@
 
 __version__ = "2.0.11+dev"
 
-# pylint: disable=unused-import
-# The names are imported here to shorten the name references to core utilities in azure.cli.core package.
+import os
+
+from knack.cli import CLI
+from knack.commands import CLICommandsLoader, CLICommand
+from knack.introspection import extract_args_from_signature, extract_full_summary_from_signature
+from knack.log import get_logger
+import six
+
+ARGCOMPLETE_ENV_NAME = '_ARGCOMPLETE'
+
+logger = get_logger(__name__)
+
+class AzCli(CLI):
+
+    def __init__(self, **kwargs):
+        super(AzCli, self).__init__(**kwargs)
+        self.session =  {
+            'headers': {},  # the x-ms-client-request-id is generated before a command is to execute
+            'command': 'unknown',
+            'completer_active': ARGCOMPLETE_ENV_NAME in os.environ,
+            'query_active': False
+        }
+
+    def get_cli_version(output):
```

Comment:
This argument is the context, ctx. Then can pass `ctx.output` to your `show_version_info_exit` method.